package.json

```diff
@@ -13,7 +13,8 @@
     "eslint-plugin-prettier": "2.4.0",
     "materialize-css": "0.100.2",
     "prettier": "1.8.0",
-    "sass-lint": "1.12.1"
+    "sass-lint": "1.12.1",
+    "snyk": "1.69.3"
```

Comment:
Can the update to the yarn.lock file be included?

https://yarnpkg.com/lang/en/docs/yarn-lock/

It should auto-generate based on changes to package.json when you run `yarn install`